.ci/setup-toolchain.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -e
+
+# Toolchain Configuration
+TOOLCHAIN_REPO="https://github.com/riscv-collab/riscv-gnu-toolchain"
+TOOLCHAIN_VERSION="2025.07.16"
```

Comment:
Validate the latest prebuilt toolchain: `2025.10.18`
https://github.com/riscv-collab/riscv-gnu-toolchain/releases